components/NavBar.js

```diff
@@ -46,11 +58,11 @@ export default function NavBar() {
 
         {(sharedState.currentCourse === course.id) &&
         <div className={styles.courseButtons}>
-          <Link href='/moduleDashboard'>
+          {/* <Link href='/moduleDashboard'>
             <div className= {styles.module}>
```

Comment:
Looks like this can be deleted now since we decided allmodules is redundant with the course